cms/static/js/views/course_outline.js

```diff
@@ -458,6 +458,43 @@ function(
             event.stopPropagation();
         },
 
+        closeManageTagsDrawer(drawer, drawerCover) {
+            $(drawerCover).css('display', 'none');
+            $(drawer).empty();
+            $(drawer).css('display', 'none');
+            $('body').removeClass('drawer-open');
+        },
+
+        openManageTagsDrawer(event) {
+            const drawer = document.querySelector("#manage-tags-drawer");
+            const drawerCover = document.querySelector(".drawer-cover")
+            const article = document.querySelector('[data-taxonomy-tags-widget-url]');
+            const taxonomyTagsWidgetUrl = $(article).attr('data-taxonomy-tags-widget-url');
+            const contentId = this.model.get('id');
+
+            // Add handler to close drawer when dark background is clicked
+            $(drawerCover).click(function() {
+                this.closeManageTagsDrawer(drawer, drawerCover);
+            }.bind(this));
+
+            // Add event listen to close drawer when close button is clicked from within the Iframe
+            window.addEventListener("message", function (event) {
+                if (event.data === 'closeManageTagsDrawer') {
+                    this.closeManageTagsDrawer(drawer, drawerCover)
+                }
+            }.bind(this));
+
+            $(drawerCover).css('display', 'block');
+            // xss-lint: disable=javascript-jquery-html
+            $(drawer).html(
+                `<iframe src="${taxonomyTagsWidgetUrl}${contentId}" frameborder="0" style="width: 100%; height: 100%;"></iframe>`
+            );
```

Comment:
Please focus() on the drawer after you open it. Currently the user's focus stays on the darkened page in the background and one has to press TAB dozens of times before it selects contents of the drawer. So to make it keyboard accessible we need to move the focus onto the drawer proactively. It's also best practice to "trap" the focus in the modal so pressing TAB just cycles the contents of the modal and never tabs onto some elements in the parent page, but that may be tricky to do in the old UI. It should be trivial in the new React UI if we implement the drawer with a Paragon `<Modal>` element in the future. (So we can do that later once the whole UI is moved to an MFE).

Reply:
Got it, I've added the auto focus on drawer opened functionality as suggested.